Model/Transaction.php

```diff
@@ -36,6 +36,8 @@ class Transaction extends MultiLanguageModel
     const STATE_CLOSED = 'closed';
     const STATE_ERROR = 'error';
 
+    const WIRETRANSFER = 'wiretransfer';
```

Comment:
Please move this to the **POI/PIA base class** as it should not be needed for any other payment method.

Reply:
I would also prefer to call it `$PAYMENT_METHOD_WIRETRANSFER` instead if possible.

Reply:
👍